### PR TITLE
[Editor]Update  particle's inspector and material's inspector 

### PR DESCRIPTION
**Pull request by chichinohaha**
Re: cocos-creator/3d-tasks#

Changelog:
add readonly to ui-prop

### CODE REVIEW

File: editor/inspector/assets/material.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ exports.$ = {
     section: 'section',
     asyncLoadAssets: '.asyncLoadAssets',
     pass: '.pass',
-
+    header: '.header',
+    section: '.section',
```
Comment (arsen2010) [inline editor/inspector/assets/material.js:36]: There's already a section up there

File: editor/inspector/assets/material.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ exports.$ = {
     section: 'section',
     asyncLoadAssets: '.asyncLoadAssets',
```
Comment (arsen2010) [inline editor/inspector/assets/material.js]: this is useless

File: editor/inspector/assets/material-header.js [hunk 3]
```diff
@@ -32,6 +32,9 @@ exports.$ = {
 };
 
 exports.methods = {
+    hideAllContent (hide) {
+        this.$.container.style= hide ? "display:none" : '';
```
Comment (VisualSJ) [inline editor/inspector/assets/material-header.js]: Please use single quotes

**Review by VisualSJ — CHANGES_REQUESTED** [review]

File: editor/inspector/assets/material-header.js (hunk 3 above)
Comment (chichinohaha) [reply editor/inspector/assets/material-header.js]: fixed

File: editor/inspector/assets/material.js (hunk 1 above)
Comment (chichinohaha) [reply editor/inspector/assets/material.js:36]: fixed

**Review by arsen2010 — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**